css/index.css

```diff
@@ -101,10 +101,100 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes .box1 {
```

Comment:
nice use of selectors
However an alternative way of doing this, to avoid repeating the same block of code for all boxes is to style the **.boxes** class which is common to all of them, then give the individual boxes their unique colors only